pkg/scheduler/util/utils.go

```diff
@@ -109,3 +109,25 @@ func GetPodAntiAffinityTerms(podAntiAffinity *v1.PodAntiAffinity) (terms []v1.Po
 	}
 	return terms
 }
+
```

Comment:
defaultpodtopologyspread will be deprecated soon, so add a todo to move this function to node_tree.go once this happens.

Reply:
sure

Reply:
@ahg-g 
updated